packages/commercetools/composables/__tests__/useNewsletter/useNewsletter.spec.ts

```diff
@@ -0,0 +1,51 @@
+import useNewsletter from '../../src/useNewsletter';
+
+jest.mock('@vue-storefront/core', () => ({
+  useNewsletterFactory: (params) => () => params
+}));
+
+describe('[commercetools-composables] useNewsletter', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('loads newsletter data', async () => {
+    const { load } = useNewsletter() as any;
+    const defaultNewsletterData = {
+      woman: false,
+      man: false,
+      kids: false
+    };
+
+    const loadedNewsletter = await load({});
+
+    expect(loadedNewsletter).toStrictEqual(defaultNewsletterData);
```

Comment:
Here you use `toStrictEqual`, but below you use `toMatchObject`. If both are used for the same purpose, I'd suggest using `toMatchObject` because it better describes what we are expecting.

Reply:
Additionally, as `load` shouldn't return any data, we should be comparing `defautNewsletterData` and `newsletter` object from `useNewsletter` composable.